index.html

```diff
@@ -27,7 +27,7 @@ <h1>Сколько я должен за обед в О'Хулиганс</h1>
 		<section class="drink">
 			Я выпил на
 			<span id="drink" contenteditable="true">200</span>₽,
-			а со скидкой в <span id="drink__discount" contenteditable="true">10</span>%
+			а со скидкой в <span id="drink__discount" contenteditable="true">20</span>%
```

Comment:
Wait, is it for everyone? Or only when you have @krigstask with you?

Reply:
@RB-Lab, that's my discount card only.